corehq/apps/commtrack/tests/test_locations.py

```diff
@@ -87,6 +87,43 @@ def test_archive_flips_sp_cases(self):
         sp = SupplyPointCase.get(sp.case_id)
         self.assertFalse(sp.closed)
 
+    def test_full_delete(self):
+        test_loc = make_loc(
+            'test_loc',
+            type='state',
+            parent=self.user.location
+        )
+        test_loc.save()
+
+        original_count = len(list(Location.by_domain(self.domain.name)))
```

Comment:
Is this part necessary?  The rest of the test assumes there's only the user's location and test_loc.